module/os/linux/zfs/vdev_file.c

```diff
@@ -268,10 +268,12 @@ vdev_file_io_start(zio_t *zio)
 		zio_execute(zio);
 		return;
 	} else if (zio->io_type == ZIO_TYPE_TRIM) {
-		int mode;
+		int mode = 0;
 
 		ASSERT3U(zio->io_size, !=, 0);
+#ifdef __linux__
```

Comment:
Is the `__linux__` needed here?  This file is already part of Linux platform code where we're allowed to do platform specific things.

Reply:
The FreeBSD user space will use the linux spl for libzpool

Reply:
I see, ok this looks good.

Reply:
This line is unchanged on FreeBSD:
https://github.com/zfsonlinux/zfs/blob/master/lib/libzpool/Makefile.am#L7